spark_client/domain.py

```diff
@@ -157,24 +157,24 @@ def __init__(self, environ: Dict = dict(os.environ)):
         self.environ = environ if environ is not None else {}
 
     @property
-    def snap_folder(self) -> str:
+    def spark_folder(self) -> str:
         """Return the SNAP folder"""
-        return self.environ["SNAP"]
+        return self.environ.get("SNAP", self.environ.get("SPARK_HOME"))
 
     @property
     def static_conf_file(self) -> str:
         """Return static config properties file packaged with the client snap."""
-        return f"{self.environ.get('SNAP')}/conf/spark-defaults.conf"
+        return f"{self.spark_folder}/conf/spark-defaults.conf"
 
     @property
     def dynamic_conf_file(self) -> str:
         """Return dynamic config properties file generated during client setup."""
-        return f"{self.environ.get('SNAP_USER_DATA')}/spark-defaults.conf"
+        return f"{self.environ.get('SNAP_USER_DATA', self.environ.get('SPARK_HOME'))}/spark-defaults.conf"
```

Comment:
This should probably be the USER home in outside of snaps

Reply:
Thanks Enrico, made the change.